### PR TITLE
PIC-625 Add Username to oAuth request for community-api

### PR DESCRIPTION
**Pull request by jhevans**

### CODE REVIEW

File: src/main/resources/application-local.yml [hunk 1]
```diff
@@ -4,10 +4,10 @@ feature:
     fetch-sentence-data: true
 
 community-api:
-  base-url: http://localhost:8096
+  base-url: http://localhost:8090
```
Comment (chris-faulkner-mojdt) [inline src/main/resources/application-local.yml:7]: The 8096 / 8097 values are there to match the docker-compose for running services locally. Do you want these permanently changed ?

**Review by chris-faulkner-mojdt — DISMISSED** [review]
LGTM

File: src/main/resources/application-local.yml (hunk 1 above)
Comment (jhevans) [reply src/main/resources/application-local.yml:7]: Ah apologies, had it running against Wiremock. Might create another config for that but will revert this

**Review by chris-faulkner-mojdt — APPROVED** [review]
LGTM

**Pull request merged**